datasets/fhad/make_data.py

```diff
@@ -82,10 +90,14 @@ class Sample:
 def main():
     # Setup Paths
     root_path = Path(ROOT)
+    sys.stdout.write(f"root_path is {root_path} which is a file: {os.path.isfile(root_path)}\n\n")
     skeleton_root_path = Path(root_path, 'Hand_pose_annotation_v1')
     object_models_root_path = Path(root_path, 'Object_models')
     object_translations_root_path = Path(root_path, 'Object_6D_pose_annotation_v1_1')
     video_files_root_path = Path(root_path, 'Video_files')
+    sys.stdout.write(f"""
+    last iteration failed here\n
+    """)
```

Comment:
Oh man.  Don't use triple quotes like this.   Here is the same message:

`print('\nlast iteration failed here\n\n\n')`

or even better and more clear:

```
print()
print("last iteration failed here")
print()
print()
```